public/js/jquery.rss.js

```diff
@@ -0,0 +1,333 @@
+(function ($) {
```

Comment:
There is no header present in the original source:
https://github.com/sdepold/jquery-rss